ddl/column.go

```diff
@@ -637,6 +643,13 @@ func (w *worker) doModifyColumn(t *meta.Meta, job *model.Job, newCol *model.Colu
 		}
 	})
 
+	if newAutoRandBits > 0 {
```

Comment:
Is it possible to be less than 0?
Or the user sets it to be 0 explicitly?

Reply:
The type of newAutoRandBits is `uint64`. Setting `auto_random` to `0` is equivalent to set nothing.